### DISCUSSION TITLE
Hardware info

### DISCUSSION THREAD

**Discussion by PChild** [team401/high-key-2024#26] [General]
- Centering wheels: can id 4, positive spin pulls note in
- Intake: can id 6, negative power pulls note in, needs 100% power, beam break is hard forward limit
- Kicker: can id 5, positive power pulls notes in, beam break is hard forward limit
- Arm: can id 9, name arm
- Arm encoder: can id 13, name armEncoder

**Comment by PChild** [comment]
LEDs are on PWM channel 0, there are 30 LEDS in the strip.

All cameras are connected to the beelink, we can grab camera IDs today. @aidnem

**Reply by aidnem** [reply]
Did you use the same cameras off of HITL? I already had them set up, I know it was cameras 1, 2, and 3 (not sure which order, I think maybe 2 was FrontLeft and then FrontMiddle and FrontRight were 1 and 3 in an unknown order), and I had them calibrated as well.

**Reply by PChild** [reply]
There were two cameras on the testbed, I took both of those, they're mounted on the intake side. Shooter side is cameras 4 and 5.

**Reply by aidnem** [reply]
Interesting, I wonder where the third camera from the testbed went.
